awx/__init__.py

```diff
@@ -149,6 +150,13 @@ def manage():
     from django.conf import settings
     from django.core.management import execute_from_command_line
 
+    # enforce the postgres version is equal to 12. if not, then terminate program with exit code of 1
+    version = connection.cursor().connection.server_version
```

Comment:
Might we prefer `pg_version` because it's cached?

https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/django/db/backends/postgresql/base.py#L335

Reply:
which looks to be `connection.pg_version`

Reply:
done. i think that'll fix the unit tests from failing as well